context/default/entrypoint.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+export PRETALX_DATA_DIR="${PRETALX_DATA_DIR:-/data}"
+
+PRETALX_FILESYSTEM_LOGS="${PRETALX_FILESYSTEM_LOGS:-/data/logs}"
+PRETALX_FILESYSTEM_MEDIA="${PRETALX_FILESYSTEM_MEDIA:-/data/media}"
```

Comment:
Shouldn't these defaults match what's set in the `.env.example`?

Reply:
This was adapted from the original image and I guess they do not show the container defaults, but rather the application defaults.